package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-oidc",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Can you revert this change - I will let npm handle the version update/git tagging etc. once this is merged. Thanks!

Reply:
✔️  This is gone.